lib/manageiq/rpm_build/generate_gemset.rb

```diff
@@ -113,13 +113,20 @@ def generate_dependency_manifest
         FileUtils.mkdir(manifest_dir)
 
         shell_cmd("gem install license_finder")
+        generate_bundler_manifest
         generate_gem_manifest
         generate_npm_manifest
       end
 
+      def generate_bundler_manifest
+        where_am_i
+        FileUtils.cp(GEM_HOME.join("vmdb/manifest"), manifest_dir.join("bundler_manifest"))
+      end
```

Comment:
Not sure we need this bit, but what do you think?  This basically copies what we generated on line 97 to here.  Do we even need line 97 anymore if we move it here?

Reply:
Another option is to symlink, but I don't know if that's going to break the tarballing

Reply:
Since we have the manifest directory now, I don't think we need to keep it in gemset/vmdb directory.